lib/pushmi_pullyu/aip/solr_fetcher.rb

```diff
@@ -0,0 +1,48 @@
+require 'pushmi_pullyu'
+require 'net/http'
+require 'json'
+
+class PushmiPullyu::AIP::SolrFetcher
+
+  def self.get_permission_object_ids(noid)
```

Comment:
Why couldn't you replace these self methods (which proxy to a new instance of the class and then call the same method but just on the instance) to do something like you did in the `fedora_fetcher.rb`? 

For example:

```
PushmiPullyu::AIP::SolrFetcher.new(noid).get_permission_object_ids

# calls simple intializer which assigned noid to instance variable 
# therefore you don't need to pass it in as an argument for `get_permissions_object_ids` instance method
```
vs

```
PushmiPullyu::AIP::SolrFetcher.get_permission_object_ids(@noid)

# which calls this method

def self.get_permission_object_ids(noid)
   # Return array of ids
   new.get_permission_object_ids(noid)
 end

```